examples/llama_recipes/Dockerfile.inference

```diff
@@ -0,0 +1,25 @@
+FROM ghcr.io/pytorch/pytorch-nightly:c69b6e5-cu11.8.0
```

Comment:
is this just the latest or pinned?

Reply:
To be honest, I just ran with the version from the toy submission. Its pinned (2 month old). Have updated to the release that should coincide with the RC1 now.